core/controllers/feedback.py

```diff
@@ -215,5 +218,28 @@ def get(self, exploration_id):
 class UnsentFeedbackEmailHandler(base.BaseHandler):
     """Handler task of sending emails of feedback messages.
     This is yet to be implemented."""
+
     def post(self):
-        pass
+        user_id = self.request.get('user_id')
+        references = feedback_services.get_feedback_message_references(user_id)
+        transaction_services.run_in_transaction(
+            feedback_services.update_feedback_email_retries, user_id)
+
+        messages = {}
+        for reference in references:
+            message = feedback_services.get_message(
+                reference.exploration_id, reference.thread_id,
+                reference.message_id)
+
+            exploration = exp_services.get_exploration_by_id(
+                reference.exploration_id)
+
+            message_text = message.text
+            if len(message_text) > 200:
+                message_text = message_text[:200] + '...'
+            messages[exploration.title] = message_text
```

Comment:
But 'messages' is a dict. So if two messages come for the same exploration title, then this line would overwrite the previous message, wouldn't it?